tests/scripts/test_scripts.py

```diff
@@ -409,6 +425,40 @@ def test_train_bc_warmstart(tmpdir):
     assert isinstance(run_warmstart.result, dict)
 
 
+def test_train_sqil_main(sqil_config):
+    # NOTE: Having four different expert types as in bc might be overkill for sqil
+    run = train_imitation.train_imitation_ex.run(**sqil_config)
+    assert run.status == "COMPLETED"
+    assert isinstance(run.result, dict)
+
+
+def test_train_sqil_with_no_demonstrations_raises_value_error(tmpdir):
+    # NOTE: This test might be overkill
```

Comment:
Agreed, I don't think we need this, it's testing `_generate_expert_trajs` in the `demonstrations.py` ingredient, which is a common code path between BC and SQIL

Reply:
Removed.